dirs/dirs.go

```diff
@@ -115,6 +115,8 @@ var (
 	SnapDeviceSaveDir string
 	SnapDataSaveDir   string
 
+	NvidiaProfilesDir string
```

Comment:
we probably shouldn't define this here, but simply compute what we need in opengl.go